taxi/views.py

```diff
@@ -34,6 +35,24 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
```

Comment:
Almost the same code. Maybe you can try like this:
Change view to:
```
class ManufacturerCreateUpdateView(
    LoginRequiredMixin, generic.CreateView, generic.UpdateView
):
    model = Manufacturer
    fields = "__all__"
    success_url = reverse_lazy("taxi:manufacturer-list")

```
And then update urls due to. The same with Car
<img width="797" alt="image" src="https://github.com/mate-academy/py-taxi-service-forms/assets/65166203/9544817a-2a8c-4b1b-9204-8164482a26db">

Reply:
there is a problem I can't find a solution to. when I implement this combined view it has to have some slug in the URL or primary key to it, that I actually don't need in the create view and need especially in the update view. Does that solution with additional kwargs suggests completely rewriting the functionality or what? 'cause I didn't find any information about build-in solutions in generic views

<img width="960" alt="image" src="https://github.com/mate-academy/py-taxi-service-forms/assets/112757782/aafd6814-74e1-42f8-a56a-6b65d28a7e41">